script/plugin_tools.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ev
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  export FLUTTER_CHANGED_PACKAGES=""
+else
+  export FLUTTER_CHANGED_PACKAGES=`git diff --name-only $TRAVIS_COMMIT_RANGE | grep -o "packages/[^/]*" | sed -e "s/packages\///g" | sort | uniq | paste -s -d, -`
```

Comment:
Don't want to make @goderbauer sad. That's a good example. I will change it to take this into account.